docs/pages/tutorials/integrating-with-nextjs.md

```diff
@@ -105,7 +101,109 @@ function MyApp({ Component, pageProps, URL }) {
 }
 ```
 
-### Environmental Variable
+## Instructions - Next 14.2.4
```

Comment:
🤔 I wonder if this makes more sense to differentiate "pages" router vs "app" router